src/cr/cube/cubepart.py

```diff
@@ -1039,7 +1050,7 @@ def scale_mean(self):
 
     @lazyproperty
     def scale_median(self):
-        """ -> np.int64, the median of scales
+        """-> np.int64, the median of scales
```

Comment:
can you remove the comma from the docstring? I think it's a mistake of mine, but considering that your black touches this line ... :D